src/simulation/thermal/sensorThermal/sensorThermal.rst

```diff
@@ -0,0 +1,162 @@
+specific heat. :math:`\dot{Q}_{in}` is the electrical power consumed by the sensor.
+
+The change in temperature is solved for, and the current temperature is updated using Euler integration:
+
+.. math::
+    T_i = T_{i-1} + \dfrac{dT}{dt} \cdot dt
+
+
+Model Assumptions and Limitations
+---------------------------------
+
+This code makes the following assumptions:
+
+- The sensor is a flat plate with an insulated backing (i.e. there is no conduction from the sensor to the spacecraft).
+- The Earth's albedo does not impact the temperature.
+- All of the electrical power is converted to heat. The heat is conducted throughout the sensor instantaneously.
+
+
+User Guide
+----------
+
+Default Parameters
+~~~~~~~~~~~~~~~~~~
+
+This module has several parameters that are set to default values, but may be changed as shown in the Module Setup
+section below. These default parameters are as follows:
+
+.. list-table:: Default Module Parameters
+    :widths: 25 50 25
+    :header-rows: 1
+
+    * - Parameter
+      - Description
+      - Default Value
+    * - nHat_B
+      - Required parameter. Normal vector of the sensor face in the spacecraft body frame. Set with `setSensorParameters().`
+      - [0.0, 0.0, 0.0]
+    * - sensorArea
+      - Required parameter. Area of the sensor. Set with `setSensorParameters().`
+      - -1
+    * - sensorAbsorptivity
+      - Required parameter. Absorptivity coefficient of sensor. Set with `setSensorParameters().`
+      - -1
+    * - sensorEmissivity
+      - Required parameter. Emissivity coefficient of sensor. Set with `setSensorParameters().`
+      - -1
+    * - sensorMass
+      - Required parameter. Mass of the sensor. Set with `setSensorParameters().`
+      - 1.0 kg.
+    * - sensorSpecificHeat
+      - Required parameter. Specific heat of the sensor. Set with `setSensorParameters().`
+      - 890 J/kg/K (aluminum).
+    * - T_0
+      - Optional parameter. Initial temperature of the sensor. Set directly.
+      - 0.0 degrees Celsius.
+    * - sensorPowerDraw
+      - Optional parameter. Power draw of the sensor. Set directly.
+      - 0.0 W.
+
+
+Module Setup
+~~~~~~~~~~~~
+
+The temperature module is created in python using:
+
+.. code-block:: python
+    :linenos:
+
+    sensorThermalModel = sensorThermal.SensorThermal()
+    sensorThermalModel.ModelTag = 'sensorThermalModel'
+
+A sample setup is done using:
+
+.. code-block:: python
+    :linenos:
+
+    nHat_B = [0, 0, 1]  # Body-frame vector of the normal face of the plate
+    sensorArea = 1.0  # m^2
+    sensorAbsorptivity = 0.25  # unitless
+    sensorEmissivity = 0.34  # unitless
+    sensorMass = 2.0  # kg
+    sensorSpecificHeat = 890  # J/kg/K
+    sensorPowerDraw = 30.0   # Watts
+    sensorThermalModel.T_0 = 0   # [Celsius]
+    sensorThermalModel.setSensorParameters(nHat_B, sensorArea, sensorAbsorptivity, sensorEmissivity, sensorMass, sensorSpecificHeat)
```

Comment:
:large_blue_circle: This example code could be less verbose if it used named args for the call. E.g. `sensorThermalModel.setSensorParameters(nHat_B-[0, 0, 1], sensorArea=1.0, sensorAbsorptivity=0.25, 
 and so on...`

Reply:
Again, I think the common way to set BSK module parameters is to do 
```
sensorThermalModel.sensorArea = ...
sensorThermalModel. sensorSpecificHeat = ...
sensorThermalModel. sensorMass = ...
```
and so on.  I don't see a value for this `setSensorParameters()` method.

Reply:
Resolved in previous comment.